java/ingest/ingest-runner/src/main/java/sleeper/ingest/job/IngestJobQueueConsumerRunner.java

```diff
@@ -30,40 +30,68 @@
 import sleeper.configuration.properties.InstanceProperties;
 import sleeper.configuration.properties.table.TablePropertiesProvider;
 import sleeper.core.iterator.IteratorException;
+import sleeper.ingest.IngestResult;
+import sleeper.ingest.task.IngestTaskFinishedStatus;
+import sleeper.ingest.task.IngestTaskStatus;
+import sleeper.ingest.task.IngestTaskStatusStore;
+import sleeper.ingest.task.status.DynamoDBIngestTaskStatusStore;
 import sleeper.statestore.StateStoreException;
 import sleeper.statestore.StateStoreProvider;
 import sleeper.utils.HadoopConfigurationProvider;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
 
 import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+import java.util.function.Supplier;
 
 public class IngestJobQueueConsumerRunner {
```

Comment:
It looks like everything but the main method could be moved into a separate class in the ingest-core module, maybe IngestTaskRunner?